packages/auto-label/src/auto-label.ts

```diff
@@ -378,7 +448,17 @@ export function handler(app: Probot) {
       logger.info(`Skipping for ${owner}/${repo}`);
       return;
     }
-
+    // If the pull request size labeling enabled, update all labels in a repo
+    // We run it here since the scheduler runs once in a while, so we dont
+    // need to update labels on pull request change event
+    if (config?.requestsize?.enabled) {
+      await syncLabels(
+        context.octokit,
+        owner,
+        repo,
+        helper.PULL_REUEST_SIZE_LABELS
+      );
+    }
```

Comment:
non-blocking comment: it's also possible to set up multiple `app.on('schedule.repository')` hooks and they will run in parallel on the same webhook.

Reply:
No issues, it's just if we want to split to avoid having a massive function doing 2 separate things.